frontend/app/src/App.test.tsx

```diff
@@ -1735,4 +1736,19 @@ describe("handles HostCommunication messaging", () => {
 
     expect(wrapper.find("DeployButton")).toHaveLength(0)
   })
+
+  it("does not relay custom parent messages by default", () => {
+    const logErrorSpy = jest
+      .spyOn(global.console, "error")
+      .mockImplementation(() => {})
+
+    const msg = new ForwardMsg()
+    msg.parentMessage = new ParentMessage({ message: "random string" })
+    instance.handleMessage(msg)
+
+    expect(logErrorSpy).toHaveBeenCalled()
+    expect(logErrorSpy.mock.calls[0][0]).toEqual(
+      "Sending messages to the host is disabled in line with the platform security policy."
```

Comment:
nit: Maybe it's better to call this `platform policy` instead of `platform security policy` since it's only partly related to security aspects.